engine/src/index.d.ts

```diff
@@ -1961,8 +1961,20 @@ export class TourDocument {
    * @returns The tour stop index
    */
   getTourStopIndexByID(id: string): number;
+
+  /** Get the tour document as an XML string */
+  getTourXML(): string;
+
+  /** Get a representation of the tour as a blob */
+  saveToBlob(): Blob;
 }
 
+export class TourEditTab {
```

Comment:
It would be good to have some basic doc-comments for this new export and its methods.